### PR TITLE
[Mutliple components] Update components that use colors to use MDCPalette.

### LINKED ISSUE #1546: Components that use colors should use MDCPalette
### Overview
Components that specify default colors should utilize MDCPalette.  As of 25.0.0, only MaterialTextFields depends on MDCPalette directly.

### PR DESCRIPTION
**Pull request by codeman7**
All components that use colors now use MDCPalette

Closes #1546

### CODE REVIEW

**Review by willlarche — APPROVED** [review]
Thank you!

**Comment by willlarche** [comment]
Spoke in Discord with you. tl;dr

> You have to add palettes as a dependency in the pod spec too.
> MaterialComponents.podspec
> this section:   s.subspec "CollectionCells" do |ss|

**Review by willlarche — CHANGES_REQUESTED** [review]
See last comment.

**Review by willlarche — APPROVED** [review]

File: MaterialComponents.podspec [hunk 1]
```diff
@@ -174,6 +174,7 @@ Pod::Spec.new do |s|
     ss.dependency "MaterialComponents/CollectionLayoutAttributes"
     ss.dependency "MaterialComponents/Ink"
     ss.dependency "MaterialComponents/Typography"
+    ss.dependency "MaterialComponents/Palettes"
```
Comment (randallli) [inline MaterialComponents.podspec:177]: needs to be added to Activity indicator too.

**Pull request merged**